src/main/java/com/tratif/pdfgen/document/DocumentDetails.java

```diff
@@ -0,0 +1,35 @@
+package com.tratif.pdfgen.document;
+
+import java.io.File;
+import java.nio.file.Files;
+
+public class DocumentDetails {
+
+    private String content;
+
+    DocumentDetails(String content) {
+        this.content = content;
+    }
+
+    public byte[] toPdf() {
+        Runtime runtime = Runtime.getRuntime();
+        try {
+            String cmdTemplate = "wkhtmltopdf %s %s";
+
+            File html = File.createTempFile("123", ".html");
```

Comment:
right now it will work always to the same filenames -- therefore you can't generate 2 pdfs at once. Use some strategy for generating filenames

Reply:
(can be UUID based?)

Reply:
According to documentation the first sting is prefix, the second is suffix, in the middle there will be random generated number like "prefix-832947982749234-suffix".

Reply:
Right!

Reply:
but prefix should be moved to some constant -- and something better than 123 (maybe class name? or program name)